Airline_Web_API/Airlina_DB/Airlina_DB.sqlproj

```diff
@@ -67,4 +67,7 @@
     <Build Include="dbo\Tables\Tickets.sql" />
     <Build Include="dbo\Tables\TicketTypes.sql" />
   </ItemGroup>
+  <ItemGroup>
+    <RefactorLog Include="Airlina_DB.refactorlog" />
+  </ItemGroup>
```

Comment:
You've deleted file from the folder on disk, but not form the project, You'll get errors again if you try to sync the database.